# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gcl/recent-binutils.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gcl-2.6.5/o/sfaslbfd.c.orig	2004-09-25 00:00:24.804187230 +0200
+++ gcl-2.6.5/o/sfaslbfd.c	2004-09-25 00:00:46.287771708 +0200
@@ -263,7 +263,7 @@
 
     current=round_up(current,1<<s->alignment_power);
 
-    current+=s->_raw_size;
+    current+=s->rawsize; 
 
   }
   curr_size=(unsigned long)current;
@@ -288,7 +288,7 @@
 
     m=round_up(m,1<<s->alignment_power);
     s->output_section->vma=(unsigned long)m;
-    m+=s->_raw_size;
+    m+=s->rawsize;
 	     
   }
 
@@ -353,7 +353,7 @@
 					     v,0,q)) 
        FEerror("Cannot get relocated section contents\n",0);
 
-     memcpy((void *)(unsigned long)s->output_section->vma,v,s->_raw_size);
+     memcpy((void *)(unsigned long)s->output_section->vma,v,s->rawsize);
      
    }
  }
